CorsixTH/Lua/earthquake.lua

```diff
@@ -145,7 +145,18 @@ function Earthquake:tick()
   self.remaining_damage = self.remaining_damage - 1
   self.damage_timer = self.damage_timer + damage_time
 
-  -- Patient falling: work in progress, see PR 1848
+  -- The below code triggers random patient falls during an earthquake.
+  -- It is currently disabled except for debugging purposes in the config file.
+  -- Current behaviour can cause empty action queues or other undesired behaviours.
+  -- Once working, the debugging flag can be removed.
+  if not TheApp.config.debug_falling then return end
+  local hospital = self.world:getLocalPlayerHospital()
+  -- loop through the patients and allow the possibility for them to fall over
+  for _, patient in ipairs(hospital.patients) do
+    if not patient.in_room and patient.falling_anim then
+      patient:falling(false)
```

Comment:
Should there be a probability roll here?

Reply:
Hmm, "tick" function eh?  Should be a very low probability if you want to notice any of a delay, I guess.

Quick falling is perhaps more useful for testing?

Reply:
Looks like it's every quake tick (which different from a regular tick), which in essence is a damage cycle completed.

Potentially its fine for now. If a patient can't fall at one part of the quake they may move and be able to later on. I think if it becomes too uniform then we can adjust later

Reply:
I agree :)   First make it work at all before starting to fine-tune it :)